src/util/api/index.js

```diff
@@ -277,8 +277,16 @@ export const getStripeFiatPrice = ({ classId }) => {
 
 export const getLatestNFTClasses = () =>
   `${LIKECOIN_CHAIN_API}/likechain/likenft/v1/class?reverse=true`;
-export const getTopNFTClasses = () =>
-  `${LIKECOIN_CHAIN_API}/likechain/likenft/v1/ranking?ignore_list=${LIKECOIN_NFT_API_WALLET}`;
+export const getTopNFTClasses = ({ before, after }) => {
+  const qsPayload = {
+    before,
+    after,
+    ignore_list: LIKECOIN_NFT_API_WALLET,
+  };
+  return `${LIKECOIN_CHAIN_API}/likechain/likenft/v1/ranking??${querystring.stringify(
```

Comment:
Doubled `?`, redundant
Also should add default values `0` for `before` and `after` for robustness? Otherwise it's sending `?before=&after=&...` which relies on how API parses empty strings

Reply:
updated to only set keys in qs object if not undefined